internal/cmd/beta/key-pair/create/create.go

```diff
@@ -0,0 +1,162 @@
+package create
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+)
+
+const (
+	nameFlag      = "name"
+	publicKeyFlag = "public-key"
+	labelFlag     = "labels"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	Name      *string
+	PublicKey *string
+	Labels    *map[string]string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create a Key Pair",
```

Comment:
```suggestion
		Short: "Creates a Key Pair",
```

All command descriptions, both sort and long, should be in the present tense, for uniformity in the whole CLI